hack/ticketgen/README.md

```diff
@@ -1,22 +0,0 @@
-# ticketgen
```

Comment:
@rewantsoni Can you confirm that we dont need this?

Reply:
ticketgen was used in previous versions and is no longer used, we can remove it now